src/data/components/CreditCard/CreditCard.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+function CreditCard(props) {
+
+    const {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props
+
+
+  return (
+    <div style={{backgroundColor: `${bgColor}`, color: `${color}` }}>
+        <h2>{type}</h2>
+        <h1>{number}</h1>
```

Comment:
Here, on the credit card you should render "*" and only the last 4 numbers of the card